ci/clair-scan/notify.go

```diff
@@ -15,30 +15,34 @@
 package main
 
 import (
+	"bytes"
 	"fmt"
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ses"
+	"gopkg.in/gomail.v2"
 	"k8s.io/klog"
 )
 
 const (
 	// This address must be verified with Amazon SES.
 	Sender = "antreabot@gmail.com"
 
-	// Account is still in the sandbox, this address must be verified.
-	// TODO: replace with Antrea mailing list
-	Recipient = "antonin.bas@gmail.com"
-
 	// The character encoding for the email.
 	CharSet = "UTF-8"
 
 	AWSRegion = "us-west-2"
 )
 
+var Recipients = []string{
+	"antonin.bas@gmail.com",
```

Comment:
Do you think you can just monitor the mail from the two alias? Just want to see if we can avoid maintaining individual mail accounts here.

Reply:
Yes, I can remove this. It was more meant as a fallback in case our google groups start blacklisting the emails sent from AWS SES for some reason, or in case the emails end up in spam when they go through the google group distribution list (which happened to me during testing).

Reply:
done